docs/docs/1.2.0/documentation/deployments.md

```diff
@@ -10,6 +10,6 @@ The two deployments modes are:
 * [**Multi-Zone**](/docs/1.1.6/deployments/multi-zone): Kuma's advanced deployment model to support multiple Kubernetes or VM-based zones, or hybrid Service Meshes running on both Kubernetes and VMs combined.
 
 :::tip
-**Automatic Connectivity**: Running a Service Mesh should be easy and connectivity should be abstracted away, so that when a service wants to consume another service all it needs is the name of the destination service. Kuma achieves this out of the box in both deployment modes with a built-in service discovery and - in the case of the multi-zone mode - with an Ingress resource and Remote CPs.
+**Automatic Connectivity**: Running a Service Mesh should be easy and connectivity should be abstracted away, so that when a service wants to consume another service all it needs is the name of the destination service. Kuma achieves this out of the box in both deployment modes with a built-in service discovery and - in the case of the multi-zone mode - with a Zone Ingress resource and Remote CPs.
```

Comment:
```suggestion
**Automatic Connectivity**: Connectivity for a service mesh should be as automatic as possible, so that when a service consumes another service the only required information is the name of the destination service. Kuma provides connectivity with built-in service discovery. For multi-zone deployments, remote control planes and a Zone Ingress resource are also provided.
```